apps/omg_eth/test/omg_eth/ethereum_height_monitor_test.exs

```diff
@@ -77,17 +77,6 @@ defmodule OMG.Eth.EthereumHeightMonitorTest do
     assert_receive(:got_ethereum_new_height, Kernel.trunc(context.check_interval_ms * 10))
   end
 
-  test "that an ethereum_new_height event is not published when the height stalls", context do
```

Comment:
just curious, why is this test going in this PR. I can't see the connection

Reply:
I've deleted this test because we've identified with @unnawut that it was testing something that shouldn't have been implemented. namely, the event bus should push block numbers regardless if the height repeats (it would be hiding re-orgs!)